docs/appendix/troubleshooting.md

```diff
@@ -3,229 +3,101 @@ title: "Troubleshooting"
-</th>
-<td> -
-</td>
-</tr>
-</table>
-
-### Github Issues
-
-#### Error #8: Git fetch invalid
-
-<table border="solid black 1px">
-<th><strong>What/Why:
-</th>
-<td>This issue is caused by a problem with the sudo env and the user env.
-When in sudo, the RSA tokens used to connect to GitHub are removed in order to use admin credentials.
-</td>
-</tr>
-<tr>
-<th>Resolution
-</th>
-<td> -
-</td>
-</tr>
-</table>
-
-#### Error #9: Error serving web
-
-<table border="solid black 1px">
-<th><strong>What/Why:
-</th>
-<td> -
-</td>
-</tr>
-<tr>
-<th>Resolution
-</th>
-<td> -
-</td>
-</tr>
-</table>
-
-## Application and Plugin Conflicts
-
-### Mac Applications
+
+This document lists common errors encountered when working with `viam-server` and the [Viam app](https://app.viam.com), and provides simple steps to resolve them.
+While many common issues and their possible resolutions are presented here, this list is not comprehensive.
+
+If you have encountered an error that is not listed here, we'd love to hear from you on our [Community Discord](https://discord.gg/viam)!
+Please post the error message you received along with how you were able to trigger it and we'll see if we can help.
+
+## Common Installation Errors
+
+### The authenticity of host 'hostname.local' can't be established
+
+- **Description:** When following our [installation guides](/installation/), you will likely encounter this message the first time you try to make an `ssh` connection to your newly-imaged {{< glossary_tooltip term_id="board" text="board" >}}.
+   This is expected: `ssh` is advising you that it has not yet connected to this address, and prompts you for how to proceed.
+
+- **Solution:** The message will ask `Are you sure you want to continue connecting?`.
+   Type `yes` and then press the return key to continue with the connection.
+   You should receive a successful confirmation message similar to: `Warning: Permanently added 'hostname.local' to the list of known hosts.`
+   This is only required for the first `ssh` connection you make to a newly-imaged board.
+
+### ssh: connect to host hostname.local port 22: Host is down
+
+- **Description:** Your computer is not able to connect to the {{< glossary_tooltip term_id="board" text="board" >}} through `ssh`.
+
+- **Solution:** Ensure that both your computer and the board itself are connected to the internet, and verify each of the following:
+  - If you are on a Windows computer, be sure that you do not have an outgoing firewall preventing `ssh` connections over port `22`.
+  - Your `ssh` connection string should resemble the following: `ssh username@hostname.local`.
+    Be sure that you match hostname, username, and password exactly to what you initially configured when imaging your board.
+  - If you are still unable to connect, try power cycling the board and trying your connection again.
+  - If that fails, try re-imaging your board following the [installation guide](/installation/) appropriate for your board.
```

Comment:
If you do that and use the same hostname there's an ssh issue where you'll have to manually remove the hostname form the hosts file - we should probably cover that too then :)

Reply:
Shouldn't need to edit `known_hosts` in this case, but you would be prompted to confirm if you chose a different hostname for the second imaging (or if you used IP like `andf@123.45.67.89` and the IP address has changed):

```
laptop:~ user-local$ ssh user-pi@my-pi-image-2.local
The authenticity of host 'my-pi-image-2.local ' can't be established.
ED25519 key fingerprint is SHA256:ABCABCABCABCABCABCABCABCABCABCACB.
This host key is known by the following other names/addresses:
    ~/.ssh/known_hosts:4: my-pi-image-1.local
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added 'my-pi-image-2.local' (ED25519) to the list of known hosts.
```

For most cases, the hostname would be the same, and the key could be reused seamlessly without user interaction.